docker/standalone-ha.xml

```diff
@@ -95,15 +93,20 @@
                     <named-formatter name="COLOR-PATTERN"/>
                 </formatter>
             </console-handler>
+            <periodic-rotating-file-handler name="FILE" autoflush="true">
```

Comment:
Thanks for the review, will remove. This came in as part of `distribution/server-dist/target/keycloak-3.2.0.Final/standalone/configuration/standalone-ha.xml`